src/hooks/useFlowChartState.ts

```diff
@@ -35,6 +35,11 @@ export interface CtlManifestType {
   layout: ReactGridLayout.Layout;
 }
 
+export type API_TYPE = {
```

Comment:
Why is this in here? it's only used in one place, `FlowChartServices.ts`.
Also, type names should be `PascalCase` not `ALL_CAPS`